tests/integration/test_review.py

```diff
@@ -1,50 +0,0 @@
-import pytest
```

Comment:
why was this file deleted?

Reply:
this was discussed with Richard here https://labelbox.slack.com/archives/C02F8TKT7RU/p1689174866073139